cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -19,7 +19,7 @@
 
 OOZIE_LIB_PATH='/usr/hdp/current/oozie'
 OOZIE_CLIENT_PATH='/usr/hdp/current/oozie-client'
-OOZIE_SERVER_PATH='/usr/hdp/2.2.0.0-2041/oozie-server'
+OOZIE_SERVER_PATH='/usr/hdp/2.2.0.0-2041/oozie/oozie-server'
```

Comment:
We can indeed, just that the HDP version is hard coded in various components throughout the code base. I will make the change here though